notification/slack/actions.go

```diff
@@ -0,0 +1,179 @@
+package slack
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/slack-go/slack"
+	"github.com/target/goalert/alert"
+	"github.com/target/goalert/config"
+	"github.com/target/goalert/permission"
+)
+
+// Handler responds to API requests from Slack
+type Handler struct {
+	c Config
+}
+
+// Payload represents the relevant payload information sent from Slack
+type Payload struct {
+	ResponseURL string `json:"response_url"`
+	Actions     []Action
+	Channel     slack.Channel
+}
+
+// Action represents the information given from an action event within Slack
+// e.g. clicking to acknowledge an alert from slack
+type Action struct {
```

Comment:
Is it necessary to export these as public?

Reply:
looks like these aren't needed anymore- removed